modules/in_band_dtmf/in_band_dtmf.c

```diff
@@ -226,7 +226,7 @@ static int in_band_dtmf_send(struct re_printf *pf, void *arg)
 
 	st = encs.head->data;
 	/* Sample count for time span of 0.1s */
-	sample_count = 2 * 0.1f * st->srate;
+	sample_count = (size_t)(2 * 0.1f * st->srate);
```

Comment:
I am not sure why the time span is 0.1 second. Anyway numerically better:
```c
sample_count = (size_t) (2 * st->srate / 10);
```

Reply:
Is the 2 because of the sample size? If so then the variable name is unsuitable. Should it be `nbytes`?